eva-accession-clustering-automation/README.md

```diff
@@ -8,38 +8,29 @@
 ## Cluster multiple assemblies
 The clustering automation script have the following parameters:
 * **source\*:** The possible sources are Mongo or VCF
-* **asm-vcf-prj-list:** Is a list of one or many assembly#vcf#project combinations. This is required is the source is VCF
+* **asm-vcf-prj-list:** Is a list of one or many assembly#vcf#project combinations. This is required if the source is VCF
 * **assembly-list:** Is a list of assemblies to process. This is required if the source is Mongo
-* **private-config-file\*:** Private configuration file with sensitive information like internal paths or tokens
-* **private-config-xml-file\*:** Maven settings.xml file with the profiles that hold database connection data
+* **github-token:** Token from github if the EVA settings configuration file should be used
```

Comment:
I have an issue with this. The very reason we pass around config files is because we don't want sensitive credentials being captured in command history. "Github token" is sort of the crown-jewel of credentials (because using it, all other credentials can be unlocked) and we shouldn't capture it in the command history.

Reply:
That's a vey good point I did not think about. I second this comment.

Reply:
Actually thinking about it, I'm not even sure that downloading from Github is a good idea at all. The private repo should be deployed in the cluster separately and the file path provided to command line (or environment variable)